CodePush.m

```diff
@@ -100,16 +109,13 @@ + (void)setUsingTestConfiguration:(BOOL)shouldUseTestConfiguration
 }
 
 /*
- * This is used to clean up all test updates. It can only be used
- * when the testConfigurationFlag is set to YES, otherwise it will
- * simply no-op.
+ * WARNING: This cleans up all downloaded and pending updates.
  */
-+ (void)clearTestUpdates
++ (void)clearUpdates
 {
-    if ([CodePush isUsingTestConfiguration]) {
-        [CodePushPackage clearTestUpdates];
-        [self removePendingUpdate];
-    }
+    [CodePushPackage clearUpdates];
```

Comment:
Hmm, I thought [[CodePush bundleURL] and in turn [CodePush clearUpdates] never gets called in debug mode](https://github.com/Microsoft/react-native-code-push/blob/clear-updates-on-new-binary/CodePush.m#L229)?